### PR TITLE
Pip installation

### PR DESCRIPTION
**Pull request by mdbrnowski**
Solves #17

### CODE REVIEW

**Review by radoslawrolka — APPROVED** [review]
lgtm

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    long_description = f.read()
+
+setup(
+    name='bitalg',
+    description='Laboratory classes in the Geometric Algorithms course for Computer Science students at the AGH University.',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    author='AGH BIT Student Scientific Group',
+    packages=['bitalg'],
+    python_requires='>=3.11'
```
Comment (Rusiek) [inline setup.py:13]: Requiring users to have a version >=3.11 is too intrusive, consider changing to >=3.8

File: bitalg/tests/test_core.py
Comment (Rusiek) [inline bitalg/tests/test_core.py:1]: Adding pip's ability to install should not change anything other than possible library imports. If this was added accidentally then probably by pull without the --rebase option, if intentionally then this should be broken into a separate commit

File: bitalg/tests/test1.py
Comment (Rusiek) [inline bitalg/tests/test1.py:1]: Adding pip's ability to install should not change anything other than possible library imports. If this was added accidentally then probably by pull without the --rebase option, if intentionally then this should be broken into a separate commit

**Review by Rusiek — CHANGES_REQUESTED** [review]
- Change the required Python version
- Add changes due to path changes only

File: setup.py (hunk 1 above)
Comment (mdbrnowski) [reply setup.py:13]: From the very beginning we required Python version 3.11. https://github.com/aghbit/Algorytmy-Geometryczne/blob/master/alg-geo.txt#L70C1-L70C76

Comment (Rusiek) [reply setup.py:13]: We proposed an environment with version 3.11, we did not require it

**Comment by mdbrnowski** [comment]
@Rusiek done

**Review by Rusiek — APPROVED** [review]
